without this patch mutt crashes if any option that contains charset is set to
empty, see upstream bug http://bugs.mutt.org/3326

--- a/init.c
+++ b/init.c
@@ -1732,6 +1732,9 @@
   char *p, *q = NULL, *s = safe_strdup (val);
   int rc = 0, strict = strcmp (opt->option, "send_charset") == 0;
 
+  if (!s)
+    return rc;
+
   for (p = strtok_r (s, ":", &q); p; p = strtok_r (NULL, ":", &q))
   {
     if (!*p)
